Docs/Book/GettingStartedInPython.rst

```diff
@@ -3379,6 +3379,86 @@ These are adapted from the definitions in Gobbi, A. & Poppinger, D. “Genetic o
 | Acidic   | ``[$([C,S](=[O,S,P])-[O;H1,-1])]``                                                                                                                                     |
 +----------+------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Filtering Molecular Datasets
+*****************************
+Several sets of rules exist for estimating the liklihood of a molecule exhibiting drug like behaviour. It's worth noting that these are rules of thumb, and that many examples of approved small molecule drugs exist that disobey these rules. 
+
+Lipinski Rule of 5
+==================
+Lipinski's "Rule of 5" [#lipinski]_ was introduced to estimate the oral bioavailability of molecules. Poor absorption is likely if the molecule violates more than one of the following conditions: 
+
+* Molecular Weight <= 500 Da
+* No. Hydrogen Bond Donors <= 10
+* No. Hydrogen Bond Acceptors <= 5
+* LogP <= 5
+
+.. doctest::
+
+  >>> from rdkit import Chem
+  >>> from rdkit.Chem import Descriptors
+  >>> mol = Chem.MolFromSmiles('CC(=O)Nc1ccc(O)cc1')  # e.g. Paracetamol
+  >>> # Ro5 descriptors
+  >>> MW = Descriptors.MolWt(mol)
+  >>> HBA = Descriptors.NumHAcceptors(mol)
+  >>> HBD = Descriptors.NumHDonors(mol)
```

Comment:
In the original Lipinski paper, the definitions of acceptor (number of Ns and Os) and donor (number of N-H and O-H bonds) were a bit idiosyncratic, so the RDKit has specialized functions for calculating these:
```suggestion
  >>> HBA = Descriptors.NOCount(mol)
  >>> HBD = Descriptors.NHOHCount(mol)
```